.github/workflows/pr-ubuntu-docker.yml

```
@@ -116,3 +116,14 @@ jobs:
                 run: |
                     cd /src/build \
                     && ctest -C ${{env.BUILD_TYPE}} --verbose --output-on-failure
+
+          - name: SP Types Check
+            run: |
+              temp=$(diff ${{github.workspace}}/skyrim-platform/src/platform_se/codegen/convert-files/skyrimPlatform.ts ${{github.workspace}}/build/dist/client/Data/Platform/Modules/skyrimPlatform.ts)
+              size=${#temp}
+              if [ "$size" != 0 ]
+              then
```

I believe this should work
```suggestion
              if diff -u ${{github.workspace}}/{skyrim-platform/src/platform_se/codegen/convert-files,build/dist/client/Data/Platform/Modules}/skyrimPlatform.ts; then
```